website/docs/user-group-role/groups/manage_groups.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Manage groups
+description: "Learn how to work with groups in authentik."
+---
+
+A group is a collection of users. Refer to the following sections to learn how to create and manage groups, assign users and roles to groups, and how [permissions](../access-control/manage_permissions.md) work on a group level.
+
+## Create a group
+
+To create a new group, follow these steps:
+
+1. In the Admin interface, navigate to **Directory > Groups**.
+2. Click **Create** at the top of the Groups page.
+3. In the Create modal, define the following:
+    - name of the group
+    - whether or not users in that group will all be superusers
```

Comment:
Is "is superuser" a role? Can we mention that here?

Reply:
is superuser is a checkbox, it is a left-over from the current permission system, where being a superuser grants anyone in that group all permissions on everything